src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -306,6 +307,68 @@ public string AskUserForValue(string message, string defaultValue, bool allowEmp
             return userValue;
         }
 
+        public Dictionary<string, string> AskUserForKeyValue(Dictionary<string, string> keyValue)
+        {
+            keyValue ??= new Dictionary<string, string>();
+
+            if (keyValue.Keys.Count == 0)
+            {
+                AskToAddKeyValuePair(keyValue);
+                return keyValue;
+            }
+
+            const string ADD = "Add new environment variable";
+            const string UPDATE = "Update existing environment variable";
+            const string DELETE = "Delete environment variable";

```

Comment:
These prompts strictly refer to environment variables but are declared as constants inside the generic method to ask for key-value pairs. These message prompts should be parameters in the method signature and will change depending on the context of the key-value pairs.

Reply:
Done